#############################################################
#                                                           #
# Copyright (c) 2021, University of California, Los Angeles #
#                                                           #
# Authors: Kenny J. Chen, Brett T. Lopez                    #
# Contact: kennyjchen@ucla.edu, btlopez@ucla.edu            #
#                                                           #
#############################################################

dlo:
  version: 1.3.1

  adaptiveParams: true

  odom: false
  imu: true
  gravityAlign: true
  trajectoryOptimization: false

  mode: 1 # 0: LOCALIZATION, 1: SLAM
  mapName: ""
  mapDirectory: "/mnt/usb_data/site_data/maps"

  odomNode:
    map_frame: map
    odom_frame: odom
    child_frame: base_link
    lidar_frame: velodyne_frame

  mapNode:
    publishFreq: 3.0
    leafSize: 0.25
    mapPrefix: exr2
